Cmdr/Shared/Registry.lua

```diff
@@ -215,6 +215,20 @@ end
 
 Registry.GetCommandsAsStrings = Registry.GetCommandNames
 
+--- Returns and array of all registered command groups
+function Registry:GetGroups()
+	local groups = {}
+
+	for _, group in pairs(Registry:GetCommands()) do
+		if not table.find(groups, group.Group) then
+			table.insert(groups, group.Group)
+		end
+	end
+	table.sort(groups)
+	
+	return groups
+end
+
```

Comment:
I think we should remove this function as it isn't providing any new information. We can derive this from the command list if it is needed. With the updated implementation I described above, we don't need a list of groups anyways.